src/components/Login/Login.js

```diff
@@ -31,14 +30,17 @@ function Login({ navigation }) {
     shallowEqual
   );
 
-  const loginUser = useCallback(() => {
+  const loginUser = () => {
     dispatch(login(email, password));
-  }, [email, password, dispatch]);
+  };
 
-  const passwordChanged = useCallback(value => setPassword(value), []);
-  const emailChanged = useCallback(value => setEmail(value), []);
+  const handleEmailChange = value => {
+    setEmail(value);
+  };
 
-  navigation.setOptions({ headerShown: false });
+  const handlePasswordChange = value => {
+    setPassword(value);
+  };
```

Comment:
this can be used directly on the component on the onChangeText prop

Reply:
Done 👍